addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -333,6 +334,12 @@ def _l10n_in_edi_ewaybill_post_invoice_edi(self, invoices):
             res[invoices] = inv_res
         return res
 
+    def _l10n_in_edi_ewaybill_get_error_message(self, code):
+        if not code:
+            return ""
+        error_message = ERROR_CODES.get(code, "")
+        return error_message or _("Error message is not available, Please contact support")
```

Comment:
I thought we would add the website as well

Reply:
@jco-odoo we are using `html_escape` so we can only put non-clickable URL only and it's a rare case so we have not added a website link
ref. https://github.com/odoo/odoo/blob/15.0/addons/l10n_in_edi_ewaybill/models/account_edi_format.py#L313

Reply:
You could move the html_escape to this method? (it is what you prefer)
```suggestion
        return error_message or _("We do not know the error message for this error code. Please contact support. ")
```

Reply:
I will go with your error message suggestion.

Reply:
@xmo-odoo we fetch a new error code list and put updated PR